[JA] Fix spelling and punctuation marks

kawaemon opened the pull request:
- Added punctuation marks.
- Corrected broken markdowns.
- Corrected some of writing errors.

cl8n commented on wiki/FAQ/ja.md:
```diff
@@ -3,18 +3,18 @@
-また[太鼓モードや](/wiki/Game_Modes/osu!taiko)[CTBとは別々にスコア管理がされます](/wiki/Game_Modes/osu!catch)
+このゲームのメインとなるモードで、カーソルを動かしサークル、スライダー、スピナーの3つをタイミングよく叩くモードです。
+叩くタイミングの正確さによって300、100、50、missと4つの判定が出ます。
+また[太鼓モードや](/wiki/Game_Modes/osu!taiko)[CTBとは別々にスコア管理がされます](/wiki/Game_Modes/osu!catch)。
```
```suggestion
また[太鼓モード](/wiki/Game_Modes/osu!taiko)や[osu!catch](/wiki/Game_Modes/osu!catch)とは別々にスコア管理がされます。
```
if there is a Japanese word for "osu!catch" you can use that instead

Joehuu replied:
I don't think game modes should be translated.

kawaemon replied:
I agree.
I'll change them.

cl8n commented on wiki/FAQ/ja.md:
```diff
@@ -96,83 +96,86 @@ Cよりも下の条件でクリア
-メインメニューのSOLOを選択すればプレイできます
-ですが、一部の譜面は強制的に[太鼓モードまたは](/wiki/Game_Modes/osu!taiko)[CTBモードにされるものもあるので注意してください](/wiki/Game_Modes/osu!catch)
+メインメニューのSOLOを選択すればプレイできます。
+ですが、一部の譜面は強制的に[太鼓モードまたは](/wiki/Game_Modes/osu!taiko)[CTBモードにされるものもあるので注意してください](/wiki/Game_Modes/osu!catch)。
```
```suggestion
ですが、一部の譜面は強制的に[太鼓モード](/wiki/Game_Modes/osu!taiko)または[osu!catch](/wiki/Game_Modes/osu!catch)にされるものもあるので注意してください。
```
let's drop the "モード" when writing game mode titles. it makes sense for "太鼓モード", but "CTBモード" or "osu!catchモード" are just redundant

kawaemon commented:
Thank you clayton for suggesting!
Committed it.

kawaemon commented:
- Corrected some of writing errors
- Added about osu!mania ( but it haven't translated yet... should I delete it? )

And I'm sorry for dismissing stable review. I didn't notice these misses..

lolita-is-godddd commented on wiki/FAQ/ja.md:
```diff
@@ -96,83 +96,86 @@ Cよりも下の条件でクリア
-メインメニューのSOLOを選択すればプレイできます
-ですが、一部の譜面は強制的に[太鼓モードまたは](/wiki/Game_Modes/osu!taiko)[CTBモードにされるものもあるので注意してください](/wiki/Game_Modes/osu!catch)
+メインメニューのSOLOを選択すればプレイできます。
+ですが、一部の譜面は強制的に[osu!taiko](/wiki/Game_Modes/osu!taiko)、[osu!catch](/wiki/Game_Modes/osu!catch)または[osu!mania](/wiki/Game_Modes/osu!mania)に変更されるので注意してください。
```
It should `譜面の中には[osu!taiko](/wiki/Game_Modes/osu!taiko)、[osu!catch](/wiki/Game_Modes/osu!catch)または[osu!mania](/wiki/Game_Modes/osu!mania)専用のものがあります。` to clear meaning

lolita-is-godddd replied:
Wrong suggestion... the comment one is right

kawaemon replied:
Fixed and committed.
Thank you for suggesting!
